### PR TITLE
Show available samples in a list

### PR DESCRIPTION
**Pull request by OscarSpruit**
Buttons are cool, but not very scalable. Soo all samples are in a list now and it should be very easy to add new entries.

image

### CODE REVIEW

**Comment by OscarSpruit** [comment]
Card is not available yet, but just to show how it looks 🤫

File: example-app/src/main/java/com/adyen/checkout/example/ui/main/ComponentItem.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.adyen.checkout.example.ui.main
+
+internal sealed class ComponentItem {
+
+    abstract val text: String
+
+    data class Title(override val text: String) : ComponentItem()
+
+    sealed class Entry(override val text: String) : ComponentItem() {
+        object DropIn : Entry("Start")
```
Comment (jreij) [inline example-app/src/main/java/com/adyen/checkout/example/ui/main/ComponentItem.kt:10]: Minor: extract these into string resources

Comment (jreij) [inline example-app/src/main/java/com/adyen/checkout/example/ui/main/ComponentItem.kt:10]: Minor: extract these into string resources

File: example-app/src/main/java/com/adyen/checkout/example/ui/main/ComponentItemProvider.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.adyen.checkout.example.ui.main
+
+internal object ComponentItemProvider {
+
+    fun getComponentItems() = listOf(
+        ComponentItem.Title("Drop In"),
```
Comment (jreij) [inline example-app/src/main/java/com/adyen/checkout/example/ui/main/ComponentItemProvider.kt:6]: Same here, string resources

Comment (jreij) [inline example-app/src/main/java/com/adyen/checkout/example/ui/main/ComponentItemProvider.kt:6]: Same here, string resources

File: example-app/src/main/java/com/adyen/checkout/example/ui/main/ComponentItem.kt (hunk 1 above)
Comment (OscarSpruit) [reply example-app/src/main/java/com/adyen/checkout/example/ui/main/ComponentItem.kt:10]: Will do, hoped this would be okay in the example app 😬

**Review by jreij — APPROVED** [review]

**Pull request merged**